chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerFixtures.kt

```diff
@@ -15,7 +15,7 @@ import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 import java.util.concurrent.ConcurrentLinkedQueue
 
-const val MAX_LOG_LENGTH = 300
+const val MAX_LOG_LENGTH = 3000
```

Comment:
What do you mean? It's just used for logging within test execution. Initial state of user subscription was not fitting 300 chars hence per @mdpye advised to increase the log. We already needed to do it when earlier investigating that repeated unread count backend bug. The impact will be only within test execution and it will be good impact as the logs will contain needed information.